Chummer/frmLifestyleNuyen.cs

```diff
@@ -47,19 +51,47 @@ private void cmdOK_Click(object sender, EventArgs e)
 
         private void frmLifestyleNuyen_Load(object sender, EventArgs e)
         {
-            lblDice.Text = string.Format(GlobalOptions.CultureInfo, LanguageManager.GetString("Label_LifestyleNuyen_ResultOf"), Dice.ToString(GlobalOptions.CultureInfo));
-            nudDiceResult.Maximum = Dice * 6;
-            nudDiceResult.Minimum = Dice;
-            nudDiceResult_ValueChanged(sender, e);
+            RefreshCboSelectLifestyle();
+            RefreshCalculation(sender, e);
         }
 
         private void nudDiceResult_ValueChanged(object sender, EventArgs e)
         {
             string strSpace = LanguageManager.GetString("String_Space");
             lblResult.Text = strSpace + '+' + strSpace + Extra.ToString("#,0", GlobalOptions.CultureInfo) + ')' + strSpace + '×'
-                             + strSpace + Multiplier.ToString(_objCharacter.Options.NuyenFormat + '¥', GlobalOptions.CultureInfo)
+                             + strSpace + SelectedLifestyle.Multiplier.ToString(_objCharacter.Options.NuyenFormat + '¥', GlobalOptions.CultureInfo)
                              + strSpace + '=' + strSpace + StartingNuyen.ToString(_objCharacter.Options.NuyenFormat + '¥', GlobalOptions.CultureInfo);
         }
+
+        private void cboSelectLifestyle_SelectionChanged(object sender, EventArgs e)
+        {
+            _objLifestyle = (Lifestyle) cboSelectLifestyle.SelectedItem;
+            RefreshCalculation(sender, e);
+            lblDice.Text = string.Format(GlobalOptions.CultureInfo, LanguageManager.GetString("Label_LifestyleNuyen_ResultOf"), SelectedLifestyle.Dice.ToString(GlobalOptions.CultureInfo));
+
+        }
+
+        private void RefreshCboSelectLifestyle()
+        {
+            List<Lifestyle> lstLifestyles = _objCharacter.Lifestyles.ToList();
+
+
```

Comment:
There a reason for this much white space?

Reply:
I insert quite a lot of whitespace while writing, because that makes it easier for me somehow. Just forgot to clean up that part.